index.html

```diff
@@ -34,3 +34,223 @@ <h1 class="homepage-heading">Emmanuel <br>
+                    </div>
+                </div>
+            </div>
+
+            <div class="projects-project5"> <img src="images/Rectangle216.png" alt="" class="projects-project5"></div>
+            <div class="projects-img2">
+                <h3 class="heading-blue-background">Simple Shell Project</h3>
+                <div class="buttons">
+                    <ul>
+                        <li><button class="button1">HTML/CSS</button></li>
+                        <li><button class="button2">Ruby on Rails</button></li>
+                        <li><button class="button3">JavaScript</button></li>
+                    </div>
+                <div class=rectangle-and-arrow>
+                    <img class="rectangle" src="images/Rectangle20.png" alt="Rectangle">
+                    <div class="union">
+                    <img class="arrow" src="images/Union.png">
+                    </div>
+                    <div class="see-project">
+                        <p class="see"><a href="#">see this project</a></p>
+                    </div>
+                </div>
+            </div>
+
+            <div class="projects-project6"> <img src="images/Rectangle217.png" alt="" class="projects-project6"></div>
+            <div class="projects-img2">
+                <h3 class="heading-blue-background">Variadic functions Projects</h3>
+                <div class="buttons">                    
+                    <ul>
+                        <li><button class="button1">HTML/CSS</button></li>
+                        <li><button class="button2">Ruby on Rails</button></li>
+                        <li><button class="button3">JavaScript</button></li>
+                </div>
+                <div class=rectangle-and-arrow>
+                    <img class="rectangle" src="images/Rectangle20.png" alt="Rectangle">
+                    <div class="union">
+                    <img class="arrow" src="images/Union.png">
+                    </div>
+                    <div class="see-project">
+                        <p class="see"><a href="#">see this project</a></p>
+                    </div>
+                </div>
+            </div>
+        </section>
+        <section class="About">
+            <div class="about-container">
+                <div class="about-heading">
+                    <h1 class="heading-about">About<h1>
+                </div>
+                <div class="about-me">
+                    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ea eiusmod</p>
+                </div>
+                    <div class=rectangle-and-arrow2>
+                        <button class="about-btn"><a href="#"> Get my Cv</a></button>
+                        <div class="union">
+                        <img class="arrow2" src="images/Union (1).png" alt="union">
```

Comment:
- [ ] kindly consider adding these icons in the get my resume button and remember to make them visible  this will help you to implement exactly the provided Figma design

Reply:
I don't understand what you mean, kindly elaborate